windows-driver-docs-pr/install/inf-addservice-directive.md

```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
 
 For more information about how to specify security descriptors, see [Creating Secure Device Installations](creating-secure-device-installations.md).
 
+**RequiredPrivileges**=*privilege-name*
```

Comment:
Should look like Dependencies:
Dependencies=depend-on-item-name[,depend-on-item-name]...

so this should be (with appropriate formatting):
RequiredPrivileges=privilege-name[,privilege-name]...